### ISSUE TITLE
New ontology request: FYECO

### ISSUE THREAD

**Issue by mah11** [OBOFoundry/OBOFoundry.github.io#1677]
Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here:
http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html

## Ontology title
Fission Yeast Experimental Conditions Ontology

## Requested ID space
FYECO

## Ontology location
https://github.com/pombase/fypo/

## Contact person
Name: Val Wood
Email address: vw253 at cam.ac.uk
GitHub username: ValWood

## Issue tracker
https://github.com/pombase/fypo/issues
Use FYECO label

## Ontology license

- [ ] CC0 (public domain)
- [x] CC-BY (version 3 or later)
- [ ] Other: *please specify*

## Available ontology formats
OBO

## What domain is the ontology intended to cover?
Brief descriptions of experimental conditions relevant to fission yeast biology (and therefore also to many other microbial species)

## Related OBO Foundry ontologies
OBI

## Intended use/related projects
Used for phenotype annotations in [PomBase](https://www.pombase.org/) and [JaponicusDB](https://www.japonicusdb.org/); FYECO could be regarded as a small subset of OBI, where using OBI directly would be overkill

## Data source
https://raw.githubusercontent.com/pombase/fypo/master/fyeco.obo

## Additional comments or remarks
This ontology has been developed mainly for in-house usage by the two fission yeast MODs named above. We're registering it here primarily to avoid namespace clashes. Meeting the remaining checklist criteria would require resources we don't have at present.

@ValWood @kimrutherford @cmungall 

## OBO Foundry pre-registration checklist

To be considered for inclusion in the OBO Foundry, an ontology must meet certain requirements, as described in in [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration review checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md). To ensure you are aware of some of its key points, please review the checklist below.

You can either check a box by submitted the request first and then using the GitHub interface, or replacing the `- [ ]` by `- [X]` in the following.

- [x] I have read an understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md) 
- [ ] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there was, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [ ] My ontology has a specific release file with a version IRI and a dc:license annotation, serialised in RDF/XML.
- [x] I understand that term definitions, while not mandatory, are key to understanding the intentions of a term especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology have definitions, using the [IAO:0000115](http://www.ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [ ] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [x] For all relationship properties (Object and Data Property) I checked whether [RO](http://www.ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [ ] For the selection of appropriate annotation properties, I looked at [OMO](http://www.ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies. 
- [x] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice.

## Metadata

Please fill in the following metadata record which will be used by the OBO Foundry website. Note that the values shown are just examples, for example `yourfourletterid` could be something like `aism`, `cohm`, `mondo` (it does not have to be four letters). `your_domain_like_for_example_anatomy` could be simply `anatomy`, and the license should be whatever your actual license is. An example can be found [here](https://github.com/OBOFoundry/OBOFoundry.github.io/edit/master/ontology/amphx.md), but you really only need to fill in the metadata mentioned here.

```
id: FYECO
title: Fission Yeast Experimental Conditions Ontology
contact:
  email: vw253 at cam.ac.uk
  label: Val Wood
description: Brief descriptions of experimental conditions relevant to fission yeast biology.
domain: phenotype
homepage: https://github.com/pombase/fypo
products:
  - id: fyeco.obo
dependencies:
  - id: ro
tracker: https://github.com/pombase/fypo/issues
license:
  url: http://creativecommons.org/licenses/by/4.0/
  label: CC-BY
usages:
  - user: https://www.pombase.org/
  - user: https://www.japonicusdb.org/
    description: Used for phenotype annotations in [PomBase](https://www.pombase.org/) and [JaponicusDB](https://www.japonicusdb.org/)
```

**Comment by matentzn** [comment]
Hey @mah11 

Unfortunately, this may be a pretty tough sell here: most of the terms in FYECO should be in other OBO ontologies. Is your situation that:

- You have been using FYECO for a long time, and are worried that another ontology will occupy its namespace, causing yet another scramble to change the prefix, fixing your local infrastructure etc.
- You do not intend others to use FYECO, you merely wish to protect the prefix
- You do not have the resources at PomBase to replace the terms in FYECO by terms from other ontologies

Does this summarise the situation correctly?

**Comment by ValWood** [comment]
Hi Nico,
This does summarize the situation. Ideally we would migrate the 381 terms to another ontology, but we  I don't have time to do the requests right now, and we need all of the terms in one place before we can migrate. So if anyone is happy to ingest FYECO for us, we can easily make the switch.

External users should at least be able to trace the conditions if they really want to, although for non-PomBase users the conditions like (agar plates, liquid culture, standard temperature, media etc) are not really critical for the interpretation of the phenotype ( some databases do not even capture this detail), hence the low priority nature of this  for me right now. I see it as a temporary solution.

**Comment by matentzn** [comment]
Alright, in this case, rather than adding FYECO to OBO foundry, I would suggest the following:

1. We register the FYECO at https://bioregistry.io/registry/. We don't accept any prefix requests in any case anymore which conflict with bioregistry, so the prefix is safe.
2. We do not add FYECO to OBO foundry, as this will confuse users with yet another ID space covering terms that do should live in OBI, CHEBI and PATO.

Would this work for you? it will give you time to migrate without putting the burden on you that you have to (this could be done as part of another grant), and you don't need to worry anyone ever taking up this space.

**Comment by ValWood** [comment]
That sounds like a good solution!

**Comment by matentzn** [comment]
Here it is:

https://github.com/biopragmatics/bioregistry/issues/268

If you are happy with this,let me us know :)

**Comment by zhengj2007** [comment]
Discussed on 12-14-2021 call. Thank you for applying for OBO registration. But we think to register to bioregistry is the best solution. If you want, we'd like to help you to submit the terms you need to existing OBO Foundry ontologies.
